cmd/gokey/main.go

```diff
@@ -79,20 +83,43 @@ func genKey(seed []byte, w io.Writer) {
 	}
 }
 
-// TODO: parametrize size
-// generates raw 32 bytes
 func genRaw(seed []byte, w io.Writer) {
+	if isInvalidLength(length) {
+		log.Fatalln("Invalid Length")
+	}
+	if !isFlagPassed("l") {
```

Comment:
and this - let's enforce it before calling this helper

Reply:
Resolved in f9832b1